app/models/users/suspended.rb

```diff
@@ -0,0 +1,20 @@
+module Users
+  class Suspended < ApplicationRecord
```

Comment:
[Question] Why not something like `SuspendedUsername` with a link back to the originating User?

Reply:
We only add a record here when a user gets deleted, so there is no originating user.

Reply:
Re the model name: we namespace some rarely used models (e.g. `Users::GdprDeleteRequest`) in order to keep the top-level model namespace a bit more orderly. I could call it `Users::SuspendedUsername` though if you'd prefer that.

Reply:
Ah, here I was thinking the user model might stick around in a logical delete situation, but that's not what we're doing.